src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -157,42 +163,71 @@ class AccountControllerTest @Autowired constructor(
                 "https://github.com"
             )
 
-            mockMvc.post("/accounts/new") {
-                contentType = MediaType.APPLICATION_JSON
-                content = objectMapper.writeValueAsString(
-                    mapOf(
-                        "name" to noWebsite.name,
-                        "email" to noWebsite.email,
-                        "password" to noWebsite.password,
-                        "bio" to noWebsite.bio,
-                        "birthDate" to noWebsite.birthDate,
-                        "photoPath" to noWebsite.photoPath,
-                        "linkedin" to noWebsite.linkedin,
-                        "github" to noWebsite.github
-                    )
-                )
+            val accountPart = MockPart("account", noWebsite.toJson().toByteArray())
```

Comment:
I don't think this is the same. websites will be serialized as `[]`, so the test is useless. That's why it previously used mapOf